pyproject.toml

```
@@ -0,0 +1,51 @@
+[metadata]
+license_file = "LICENSE"
+description-file = "README.md"
+
+
+[tool.pytest.ini_options]
+addopts = '-m "not conda"'
+markers = [
+    "conda: test related to conda forge distribution"
+]
+
+[tool.black]
+# https://github.com/psf/black
+line-length = 120
+exclude = "(.eggs|.git|.hg|.mypy_cache|.venv|_build|buck-out|build|dist)"
+
+
+[tool.ruff]
+line-length = 127
```

Shouldn't this match `black`'s config?
```suggestion
line-length = 120
```

---

it should; I just took the same config as it was for flake8